sound/soc/sof/sof-priv.h

```diff
@@ -565,6 +568,12 @@ struct snd_sof_dev {
 	 */
 	bool dspless_mode_selected;
 
+	/*
+	 * Flag set when DSP ops testing is enabled. When this is set normal audio functionality
+	 * including DSPless mode is disabled.
```

Comment:
I really believe this is only relevant IF the DSPless mode is NOT enabled. It's more logical to me...